packages/jest-snapshot/src/__tests__/inline_snapshots.test.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+jest.mock('fs');
+jest.mock('prettier');
+
+const fs = require('fs');
+const path = require('path');
+const prettier = require('prettier');
+
+const {saveInlineSnapshots} = require('../inline_snapshots');
+
+const writeFileSync = fs.writeFileSync;
+const readFileSync = fs.readFileSync;
+const existsSync = fs.existsSync;
+const statSync = fs.statSync;
+const readdirSync = fs.readdirSync;
+beforeEach(() => {
+  fs.writeFileSync = jest.fn();
+  fs.readFileSync = jest.fn();
+  fs.existsSync = jest.fn(() => true);
+  fs.statSync = jest.fn(filePath => ({
+    isDirectory: () => !filePath.endsWith('.js'),
+  }));
+  fs.readdirSync = jest.fn(() => []);
+
+  prettier.resolveConfig.sync.mockReset();
+});
+afterEach(() => {
+  fs.writeFileSync = writeFileSync;
+  fs.readFileSync = readFileSync;
+  fs.existsSync = existsSync;
+  fs.statSync = statSync;
+  fs.readdirSync = readdirSync;
+});
+
+test('saveInlineSnapshots() replaces empty function call with a template literal', () => {
+  const filename = path.join(__dirname, 'my.test.js');
+  fs.readFileSync = jest.fn(() => `expect(1).toMatchInlineSnapshot();\n`);
+
+  saveInlineSnapshots(
+    [
+      {
+        frame: {column: 11, file: filename, line: 1},
+        snapshot: `1`,
+      },
+    ],
+    prettier,
+  );
+
+  expect(fs.writeFileSync).toHaveBeenCalledWith(
+    filename,
+    'expect(1).toMatchInlineSnapshot(`1`);\n',
+  );
+});
+
+test.each([['babylon'], ['flow'], ['typescript']])(
```

Comment:
We handle 1D arrays as well, so you can do `test.each(['babylon', 'flow'], 'typescript'])(`